app/controllers/pokemons_controller.rb

```diff
@@ -0,0 +1,33 @@
+class PokemonsController < ApplicationController
+  def index
+    @pokemon = Pokemon.all
+  end
+
+  def show
+    @pokemon = Pokemon.find[params[:id]]
+  end
+
+  def new
+    @pokemon = Pokemon.new
+  end
+
+  def create
+    pokemon = Pokemon.new(pokemon_params)
+    pokemon.user = current_user
+    if pokemon.save
+      redirect_to pokemons_path
+    else
+      render :new
+    end
+  end
+
+  # add edit, update and destroy
+
+  private
+
+  def pokemon_params
+    params.require(:pokemon). permit(:name, :description, :location, :price, :user_id)
```

Comment:
1. Remove space after `params.require(:pokemon).` 
2. in `permit()` if the user is signed in and adding a pokemon, we don't need it to ask him which user the pokemon is tied to, it should set the pokemon.user through the controller, like on line 16 `pokemon.user = current_user`